packages/cli/generation/ir-generator/src/examples/validateTypeExample.ts

```diff
@@ -70,18 +75,22 @@ export function validateTypeExample({
                 typeResolver,
                 exampleResolver,
                 workspace,
-                breadcrumbs
+                breadcrumbs,
+                depth
             });
         },
         undiscriminatedUnion: (rawUnion) => {
+            // console.log("validateUndiscriminatedUnionExample");
+            // console.log("validateUndiscriminatedUnionExample with Union", JSON.stringify(rawUnion, null, 2));
```

Comment:
nit: lets remove these before merging!

Reply:
Oops, should've caught these. Thanks for catching this.